packages/perseus/src/widgets/interactive-graphs/interactive-graph-validator.test.ts

```diff
@@ -172,14 +237,24 @@ describe("InteractiveGraph.validate on a segment question", () => {
 describe("InteractiveGraph.validate on an angle question", () => {
     it("marks the answer invalid if guess.coords is missing", () => {
         const guess: PerseusGraphType = {type: "angle"};
-        const rubric: PerseusInteractiveGraphRubric = createRubric({
-            type: "angle",
-            coords: [
-                [1, 1],
-                [0, 0],
-                [-1, -1],
-            ] as [Coord, Coord, Coord],
-        });
+        const rubric: PerseusInteractiveGraphRubric = {
+            graph: {
+                type: "angle",
+                coords: [
+                    [1, 1],
+                    [0, 0],
+                    [-1, -1],
+                ],
+            },
+            correct: {
+                type: "angle",
+                coords: [
+                    [1, 1],
+                    [0, 0],
+                    [-1, -1],
+                ],
+            },
+        };
```

Comment:
It really bugs me we have to duplicate this. **_I think we should probably keep it as is_** so we can move on, but:

1. `PerseusInteractiveGraphRubric` should probably be generic
2. I think a lot of these optional properties shouldn't be optional
3. This works even though I think it's technically (from a types perspective) incorrect:

``` TS
        const shared: AngleGraphCorrect = {
            type: "angle",
            coords: [
                [1, 1],
                [0, 0],
                [-1, -1],
            ],
        };
        const rubric: PerseusInteractiveGraphRubric = {
            graph: shared,
            correct: shared,
        };
```

But I'm fine moving on as long as we have some tickets for these problems.

Reply:
To Matthew's point, I don't think the `coords` need to be present on any of the `graph` objects in these tests. The `graph.coords` aren't used for scoring.

I.e. in this test, the rubric could be:

```ts
        const rubric: PerseusInteractiveGraphRubric = {
            graph: {type: "angle"},
            correct: {
                type: "angle",
                coords: [
                    [1, 1],
                    [0, 0],
                    [-1, -1],
                ],
            },
        };
```

That would solve some of the duplication.

Reply:
Removed the `coords` from each rubric's graph object. 

@handeyeco I am up for doing more work on this if it needs to be fixed while its fresh on my mind. If you would prefer to leave it for later, totally cool with that though. But, for the ones that are currently optional, they are most likely optional because the validator function has if/else statements where there is an else statement to cover when the property is undefined. If that's meant to be more of a catchall than a true option, I can just change the types to be required (I would prefer that anyway too if it's correct).

I'm not sure I understand how to make it generic, but I'll do a short bit of research to see if I can find an example.

Reply:
Here's my first attempt and what I'm testing out atm:
```
export interface PerseusInteractiveGraphRubric2<C, G> {
    correct: C;
    graph: G;
}
```

Reply:
Made the optional properties required, but might have to look back at making things generic. Will land it as it is now and come back if needed.